lib/math.js

```diff
@@ -23,6 +23,14 @@ class CryptoRandomPrefetcher {
     this.pos = end;
     return buf;
   }
+
+  [Symbol.iterator]() {
+    return {
```

Comment:
The returned object should also include `Symbol.iterator` method.

Reply:
Done.